### PR TITLE
adding secure-only SW model

### PR DESCRIPTION
**Pull request by jkrech**

### CODE REVIEW

**Comment by ReinhardKeil** [comment]
I suggest to hold back with this modification.  Devices that have TrustZone can all execute a secure-only project. IMHO this is only relevant for the CMSIS-Toolbox, but not for the PDSC specification.

**Comment by jkrech** [comment]
`Dsecure` is only used in conditions and allows to select  different components / files within a component based on the selection of the software model of the project.
It does not describe a capability/feature of a processor/device.

File: schema/PACK.xsd [hunk 1]
```diff
@@ -1185,6 +1187,8 @@
       <xs:enumeration value="Secure" />
       <!-- TZ disabled -->
       <xs:enumeration value="TZ-disabled" />
+      <!-- TZ disabled -->
```
Comment (jeromecoutant) [inline schema/PACK.xsd:1190]: Secure-only

Comment (jkrech) [reply schema/PACK.xsd]: Thanks. Fixed now.

**Comment by VGRSTM** [comment]
> This PR is parked until we have evidence that component/file filtering based on `Dsecure="Secure-only"` is required or whether using component variants specified in dedicated template projects solves this sufficiently.

I guess at least it may have an impact on linker files ... I consider this like enough to unpark.

**Review by jeromecoutant — APPROVED** [review]
This has impact in all etc/XXX.cmake files

File: schema/PACK.xsd
Comment (brondani) [inline schema/PACK.xsd:1]: See the schema check failures: date and revision should be updated.

Comment (jkrech) [reply schema/PACK.xsd:1]: fixed

**Review by brondani — APPROVED** [review]
LGTM

**Pull request merged**